src/Service.Tests/CosmosTests/SchemaGeneratorTest.cs

```diff
@@ -236,6 +236,26 @@ public void TestEmptyJsonArray()
             Assert.ThrowsException<InvalidOperationException>(() => SchemaGenerator.Generate(jsonArray, "containerName"));
         }
 
+        /// <summary>
+        /// Tests the <see cref="SchemaGenerator.Generate"/> method with an empty JSON array.
+        /// Ensures that the method correctly handles an empty input.
+        /// </summary>
+        [TestMethod]
+        public void TestEmptyJsonArrayInPayload()
+        {
+            List<JsonDocument> jsonArray = new() {
+                JsonDocument.Parse(@"{ ""name"": ""John"", ""product"": [], ""isStudent"": false, ""birthDate"": ""1980-01-01T00:00:00Z"" }")};
+            string actualSchema = SchemaGenerator.Generate(jsonArray, "containerName");
+
+            string expectedSchema = @"type ContainerName @model(name: ""ContainerName"") {
+  name: String!,
```

Comment:
so, the expectation is to skip the empty JSON array as a potential field altogether? here `product`?

Reply:
yes, because, we don't know the type of this array, it will come up if any other records have this array as non-empty.